src/grimp/exceptions.py

```diff
@@ -62,6 +62,9 @@ def __eq__(self, other):
             other.text,
         )
 
+    def __reduce__(self):
+        return SourceSyntaxError, (self.filename, self.lineno, self.text)
```

Comment:
See https://stackoverflow.com/a/36342588

Reply:
Probably worth adding a comment to explain why this is necessary.